src/vm/runtimehandles.cpp

```diff
@@ -2615,6 +2615,10 @@ void QCALLTYPE RuntimeMethodHandle::Destroy(MethodDesc * pMethod)
     // Fire Unload Dynamic Method Event here
     ETW::MethodLog::DynamicMethodDestroyed(pMethod);
 
+    BEGIN_PIN_PROFILER(CORProfilerTrackJITInfo());
```

Comment:
This event feels like it would merit a new flag rather than re-using the JIT compilation flag. Maybe COR_PRF_MONITOR_DYNAMIC_FUNCTION_UNLOADS. Conceptually the lifetime of the code which backs a method could be different than the method itself, even though today its pretty close to 1:1.

We already have a COR_PRF_MONITOR_FUNCTION_UNLOADS, which is unused, but I hesitate to re-use it because we might trigger annoying compat issues and flag bits are cheap.